src/components/ChatEntry.js

```diff
@@ -1,22 +1,36 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
-const ChatEntry = (props) => {
+const ChatEntry = ({ id, sender, body, timeStamp, liked, updateLikedEntry }) => {
+  const chatEntryClassName = id % 2 === 1 ? 'chat-entry local' : 'chat-entry remote';
+
+  const updateLiked = () => {
+    updateLikedEntry(id)
+  }
+
+  const chatEntryLiked = liked === true ? '❤️' : '🤍';
+  
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={chatEntryClassName}>
+      <h2 className="entry-name">{ sender }</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{ body }</p>
+        <p className="entry-time"><TimeStamp time={ timeStamp }/></p>
+        <button className="like" onClick={updateLiked}>{chatEntryLiked}</button>
       </section>
     </div>
   );
 };
 
 ChatEntry.propTypes = {
-  //Fill with correct proptypes
+  id: PropTypes.number,
```

Comment:
I'd recommend making `id` and `liked` required in your prop types since they are needed for the like button to work.